.analysis_options_repo

```diff
@@ -116,7 +116,7 @@ linter:
     # - prefer_interpolation_to_compose_strings # not yet tested
     - prefer_is_empty
     - prefer_is_not_empty
-    # - public_member_api_docs # this is the only difference from .analysis_options_repo
+    - public_member_api_docs # this is the only difference from .analysis_options_repo
```

Comment:
The comment was always true. We just accidentally made it _not_ a difference a while back.
See https://github.com/flutter/flutter/pull/9330